src/pages/ArticleCardDetails.jsx

```diff
@@ -24,6 +25,23 @@ const ArticleCardDetails = () => {
       console.log("No article", error);
  });
 }, [article_id])
+
+const handleUpdatedVote = (updatedVote) => {
+  setArticle((currArticle) =>({
+    ...currArticle, votes: currArticle.votes + updatedVote,
+  }));
+  setUpdatedVote((currentVote) => currentVote + updatedVote);
+
+  patchArticleVotes(article_id, updatedVote)
+  .catch((error) => {
+    console.log("Voting isn't possible, try again later", error);
```

Comment:
If I am viewing your website as a user (not a software dev 🤓 ) I will not see this message. It would be good to have the message rendered on the page, by the buttons.